src/lib.rs

```diff
@@ -255,18 +245,17 @@ impl Github<HttpsConnector<HttpConnector>> {
         A: Into<String>,
         C: Into<Option<Credentials>>,
     {
-        let connector = HttpsConnector::new(4, handle).unwrap();
-        let http = Client::configure()
-            .connector(connector)
+        let connector = HttpsConnector::new(4).unwrap();
+        let http = Client::builder()
             .keep_alive(true)
-            .build(handle);
+            .build(connector);
```

Comment:
`handle` should be removed from this function's arguments (or marked `_handle`)

Reply:
the options are to remove the handle argument or using the code i've written in the PR comment.

i would go with the breaking change and let the user use the `Github::custom` to get old behaviour.

Reply:
> i would go with removing the handle and let the user use the Github::custom to get the old behaviour.

I agree. One of the reasons I'm looking to upgrade is to remove old dependencies.